packages/@guardian/source-react-components-development-kitchen/src/components/toggle-switch/styles.ts

```diff
@@ -90,4 +93,5 @@ export const labelStyles = css`
 	display: inline-block;
 	transform: translateY(1px);
 	color: ${neutral[7]};
+	flex-shrink: 100;
```

Comment:
100 seems very large. What was the goal here with this property? There might be a different approach

Reply:
I wasn't sure what value to choose here. If I remove the flex-shrink, the button width would do down. And when flex-shrink is smaller than 50 it would still affect the button width.  I still wasn't sure if 100 is the best value. What other approach do you suggest? 

With no flex-shrink 
![image](https://user-images.githubusercontent.com/15894063/140087409-5e1756f7-de64-4131-9900-f6ff9d4f70b6.png)

With flex-shrink 30
![image](https://user-images.githubusercontent.com/15894063/140087494-6f3522a9-3d1c-4400-b203-eac06b7c9f8a.png)

With flex-shrink 50 
![image](https://user-images.githubusercontent.com/15894063/140087552-fbefa9a2-f479-48e2-9528-4b7847e0367f.png)

Reply:
You could set `flex: none` on the button, so it maintains the width you set.

Reply:
Thanks @SiAdcock it worked well with `flex: none` 🙂